Node/src/TeamsChatConnector.ts

```diff
@@ -171,17 +184,27 @@ export class TeamsChatConnector extends builder.ChatConnector {
         // Add intent message
         if (intentMessage.type == 'message') {
           var atMentions = [];
+          var botId: string = null;
+          if (re.address && re.address.bot && re.address.bot.id) {
+            botId = re.address.bot.id;
+          }
+
           intentMessage.intentText = intentMessage.text;
```

Comment:
You don't need IntentMessage at all, do you? You can store the working text in a temporary variable "strippedText" and replace "intentMessage.intentText" with "strippedText".